Chinese Translation for 1.18.2

ChuijkYahus opened the pull request.

ChuijkYahus committed:
update

Cactusstudent commented on src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/boundless.json:
```diff
@@ -0,0 +1,14 @@
+  ],
+  "effects": [
+    "将保护加成的上限提升10%",
+    "第二件盔甲则再额外提升5%",
```
```suggestion
    "第2件盔甲则再额外提升5%",
```

ChuijkYahus committed:
Update src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/boundless.json

Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

ChuijkYahus committed:
puntuation fix

ChuijkYahus committed:
Celcius symbol rollback

- ℃->°+C

ChuijkYahus committed:
mob head name standardization

ChuijkYahus committed:
change ricochet's name

- reflecting & ricochet accidentally used the same name before

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
breathtaking name change so that it fits its actual effect more

ChuijkYahus committed:
interaction toggleness

Cactusstudent committed:
小修修补《工具装置宝典》

Fix Tinker Gadgetry

ChuijkYahus committed:
Merge pull request #1 from Cactusstudent/1.18.2

小修小补

Cactusstudent committed:
Correct some chars

更改“/”为“或”，微调部分语序

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Merge branch 'ChuijkYahus:1.18.2' into 1.18.2

Cactusstudent committed:
Trans & Fix

Add Author Trans, Fix some mistakes

ChuijkYahus committed:
Merge pull request #2 from Cactusstudent/1.18.2

Correct some chars

ChuijkYahus committed:
Update

- Boundless modifier: corrected "pieces" trsl
- Chorus material: deleted "果" as the name doesn't contain "fruit"
- Blazing Bone trait: corrected desc

ChuijkYahus committed:
Update

- Selena author page: changed spaces for text layout
- Traits: standardize to "特性"

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Update

- New tooltip keys

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn

Cactusstudent committed:
Merge branch 'ChuijkYahus:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
改改改

Cactusstudent committed:
Merge branch '1.18.2' of https://github.com/Cactusstudent/TinkersConstruct into 1.18.2

Cactusstudent committed:
Apply suggestions from code review

Cactusstudent committed:
Merge branch 'ChuijkYahus:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Merge branch 'ChuijkYahus:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge pull request #3 from Cactusstudent/1.18.2

Update zh_cn

Cactusstudent committed:
Update zh_cn.json based on the latest lang file

Cactusstudent committed:
Update book based on the latest

Cactusstudent committed:
Translate & Correct
rename 弹射物伤害 to 弹射物力量
translate slings' modifier

Cactusstudent committed:
Translate part of the ency and all of the mightly

Cactusstudent committed:
Correction
Typo fix, 25% to 50%

Cactusstudent committed:
Typo Fix

Cactusstudent committed:
translate encyclopedia
correct and translate

Cactusstudent committed:
Translate all of the ency

1. translate
2. correct incorrect modifier id, firestarter to warping and springing

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
update

Cactusstudent committed:
Term change
跳跃（Springing）->飞跃

Cactusstudent committed:
Correct

Cactusstudent committed:
Terminology Changes

After Voting in Chinese Players' Community, We decide to do changes below.
You can found the voting here: https://bbs.mcmod.cn/thread-16645-1-1.html
1. 采掘等级->挖掘等级
2. 采掘速度->挖掘速度

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent committed:
采掘速度->挖掘速度

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
fix

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn based on the latest en_us

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent committed:
Fix Typo

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn based on en_us

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn.json
translate enderslime staff

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn

translate pierce modifier
translate new advancement
fix typo

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent committed:
Translate book

Cactusstudent committed:
Fix Typo

NEED REVIEW

Cactusstudent committed:
Typo Fix

Cactusstudent committed:
Update piercing.json

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn based on the latest en_us

Cactusstudent committed:
Transalte

Cactusstudent committed:
Translate & Small Fix

Airborne Accuracy -> 悬空精确度

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn
Update based on en_us, correct the descption of the advancement

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

ChuijkYahus committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update and Rename Based on the latest English File

Cactusstudent committed:
Revert "Update and Rename Based on the latest English File"

This reverts commit 8f4f096ae6fc9631982e42231f64765b8300b808.

Cactusstudent committed:
Update

Cactusstudent committed:
Update zh_cn based on the latest en_us

KnightMiner commented:
I should be done with any major changes that require new translations after that last commit, just have bug fixes and more minor features remaining before the release of the update.

Is this PR about ready to merge or would you prefer waiting until post update?

Cactusstudent commented:
> Is this PR about ready to merge or would you prefer waiting until post update?

Oh, thank you very much. I have basically completed the conversion of the file path. If possible, please merge this PR this time (Because the old translation is outdated long time) but before that, we still have some details to discuss. I think we can get these done in one day. At last, please review this before merging, to make sure there isn't any technical problem.

KnightMiner commented:
Yeah, I have been behind on handling translations for awhile. I plan on holding off on the update until I process at least this one as its been active and looks mostly complete.

Worst case if anything is not finished here when its time for the update you can open a new PR with the remaining work, as there will probably be another bug fix release in the future.

Cactusstudent committed:
Typo Fix

手杖类型 ->  黏液手杖 to keep the compatibility

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent commented:
@KnightMiner We've done all these work, looks good to us, hope there has no big bugs or typos. You can merge it. Thanks a lot for reviewing this.

Cactusstudent reviewed (approved):
lgtm，hope this pr will work successfully

Cactusstudent commented:
> > Is this PR about ready to merge or would you prefer waiting until post update?
> 
> Oh, thank you very much. I have basically completed the conversion of the file path. If possible, please merge this PR this time (Because the old translation is outdated long time) but before that, we still have some details to discuss. I think we can get these done in one day. At last, please review this before merging, to make sure there isn't any technical problem.

KnightMiner commented on src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json:
Accidental renames of protection modifiers, it should be `_protection`, nor `projectile`. Name should match the name from `modifier_id`

Same for other protection types

Cactusstudent replied:
oh, thx. It based on the en_us, so if this was wrongly named, maybe en_us also need a correction?

KnightMiner replied:
Huh, apparently english is incorrect there. Did it correctly on every other language.

KnightMiner replied:
Looking through the rest, everything seems to load fine, so I'll just merge this then rename the files after merge
